src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [querySearch, setQuerySearch] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [error, setError] = useState(false);
```

Comment:
For boolean values it is better to give names starting with is/are/can etc. so it's clear that value has boolean type.
![image](https://user-images.githubusercontent.com/96409563/195035172-64a26c55-4835-44f3-ab65-a54e22d8fd4d.png)